app/services/plans/validate_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Plans
+  class ValidateService
```

Comment:
Good to have this new logic! It will be impacted by this new PR: https://github.com/getlago/lago-api/pull/457

Let's see witch one is merge first :)